buddyboss/buddyboss-platform
# PROD-736 Fix the wrong translation issue for comment

### All Submissions:


### Changes proposed in this Pull Request:



Fixes #2960 

### How to test the changes in this Pull Request:

1. Translate word using the loco translator
2. Add a comment in activity.
3. Check translation for comment.

### Other information:
